src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -2,16 +2,23 @@
 
 // The value of component scoped variables is always defined by a global variable:
 .pf-l-toolbar {
+  padding: 0 0 2em 2em;
+
   // Toolbar Section
-  --pf-l-toolbar__section--MarginTop: var(--pf-global--spacer--md);
+  --pf-l-toolbar__section--MarginTop: var(--pf-global--spacer--xl);
 
   // Toolbar Group
+  --pf-l-toolbar__group--Padding: var(--pf-global--spacer--xl);
+  --pf-l-toolbar__group--Margin: var(--pf-global--spacer--md);
   --pf-l-toolbar__group--MarginRight: var(--pf-global--spacer--xl);
-  --pf-l-toolbar__group--MarginLeft: var(--pf-global--spacer--xl);
 
   // Toolbar Item
+  --pf-l-toolbar__item--MarginTop: var(--pf-global--spacer--md);
   --pf-l-toolbar__item--MarginRight: var(--pf-global--spacer--md);
-  --pf-l-toolbar__item--MarginLeft: var(--pf-global--spacer--md);
+
+  .pf-l-toolbar__group {
+    margin-right: var(--pf-l-toolbar__group--MarginRight);
```

Comment:
I don't think this is correct. This creates extra space between the right edge of the toolbar and the right edge of the last toolbar group. You don't want `margin-right` on the last toolbar group. I believe the way it was before was correct. Also, avoid nesting selectors unless absolutely necessary. It's implied that `.pf-l-toolbar__group` is going to be in a `.pf-l-toolbar`, so no need to nest this selector in `.pf-l-toolbar`. Nesting like this unnecessarily increases specificity.

Reply:
Reverted the code back to its initial state.